bin/mpiexecjl

```diff
@@ -61,7 +61,10 @@ fi
 SCRIPT='
 using MPI
 ENV["JULIA_PROJECT"] = dirname(Base.active_project())
-mpiexec(exe -> run(`$exe $ARGS`))
+mpiexec()
+p = mpiexec(exe -> run(pipeline(`$(exe) $(ARGS)`; stdout, stderr); wait=false))
+wait(p)
+exit(p.exitcode)
```

Comment:
Memo to self for when I'll be able to come back to this: since now we don't get a clear indication that the process failed, when the exit code is different from zero we could add a logging message like (to be improved, this is just an idea)
```julia
@error "The MPI process failed" p
```
to restore the information.